src/components/v5/shared/Button/Button.tsx

```diff
@@ -13,6 +13,7 @@ const displayName = 'v5.Button';
 
```

Comment:
because we have a lot of cases to handle is hard to read that file, maybe better solution will be create more then one button? They can depends on their destiny

Reply:
or create "base button" and try to build others button base on that

Reply:
I think the text button could be separate.
I'll do it.

Reply:
Done.